app/sass/settings/_liberator.scss

```diff
@@ -18,3 +19,6 @@ $inuit-widths-columns: (
 
 $base-font-stack: Georgia, Cambria, "Times New Roman", Times, serif;
 
+$inuit-enable-margins--large: true;
+$inuit-enable-responsive-margins: true;
+$inuit-generate-spacing-alias: s;
```

Comment:
Move this variable to `settings/_defaults.scss` and give it value of `false` as a workaround until we resolve this bug.